### PR TITLE
Went through and updated the docs source to change FreeCAD and CQFM…

### PR DESCRIPTION
**Pull request by jmwright**
…content and references to PythonOCC and CQ-editor.

I haven't worked on getting this automatically publishing via CI yet. I didn't think there was much reason to do that until the docs more accurately reflected the current state of the project.

### CODE REVIEW

**Comment by jmwright** [comment]
@adam-urbanczyk Should I set this up to be merged into your update branch instead?

File: doc/quickstart.rst [hunk 1]
```diff
@@ -149,10 +146,10 @@ Good news!-- we can get the job done with just two lines of code. Here's the cod
         .cboreHole(2.4, 4.4, 2.1)
 
     # Render the solid
-    build_object(result)
+    show_object(result)
 
 
-After pressing F2 to re-execute the model, you should see something like this:
+After clicking the Redner button to re-execute the model, you should see something like this:
```
Comment (adam-urbanczyk) [inline doc/quickstart.rst]: Redner --> Render

Comment (jmwright) [reply doc/quickstart.rst]: Thanks, I fixed that typo. I also turned on spell-checking in Sublime Text and found/fixed several others.

**Comment by adam-urbanczyk** [comment]
Thanks for updating the docs @jmwright !

**Pull request merged**